src/main/resources/templates/albums.html

```diff
@@ -9,30 +9,57 @@
     <title>My Albums</title>
 </head>
 <body>
+
+<form action="/addAlbum" method="POST">
+    <fieldset>
+        <legend>ADD ALBUM</legend>
+        <label >Album Name:</label>
+        <input type="text" name="title" required><br>
+
+        <label >Artist:</label>
+        <input type="text"  name="artist" required><br>
+
+        <label >Length (in minutes):</label>
```

Comment:
I was able to crash the app several times by writing out numbers in the Length and Song Count fields (i.e., "one" instead of 1). The easiest way to prevent that is to change the input fields to `type="number"` instead of `"text"`. (You could also do something in the back end to convert the strings to numbers, but I think that's more complicated than necessary for this project.)